airflow/providers/vertica/hooks/vertica.py

```diff
@@ -46,5 +46,41 @@ def get_conn(self) -> connect:
         else:
             conn_config["port"] = int(conn.port)
 
+        bool_options = ["connection_load_balance", "binary_transfer", "disable_copy_local", "request_complex_types", "use_prepared_statements"]
+        std_options = ["session_label", "backup_server_node", "kerberos_host_name", "kerberos_service_name", "log_path", "unicode_error", "workload", "ssl"]
```

Comment:
I carefeully looked at the parameters passed, and I think being able to set  log_path from extra is potentially dangerous and we've learned that we should carefully look at the parameters that you pass from connection definition. Users who have the rights to edit the connection should not have some additional priviledges, and there is a subtle exception here in "log_path". 

Enabling log_path to be set "freely" opens the doors to a subtle security issue, where you could override some files by setting the log path when you configure the connecion to point to a file that you want to override. I personally think that we should not allow log_path setting at all and leave it with the default "vertica_python.log" in default director

Reply:
I've made a try setting log_level without setting log_path, default directory turns to be / when runned by scheduler which make exexution fail since it doesn't have root rights. But, since vertica logging is visible in dag run logs, the log file may not be really useful, so I propose to remove the log_path parameter (in order to remove potential security issue) and pass /dev/null as log path when log_level is defined. Doing this we keep the possibility to have traces from vertica client but without the inconvient to have a log file who will contains the same informations than the dag run logs.

If it's ok for you I'll make the change in code, if not I prefer to remove the logging parameters

Reply:
Finnally I set log_path to None when log_level is set, it's prevent generation of log file and is cleaner than /dev/null

Reply:
Looks good. Just remove it also from examples :).

Reply:
done